src/types.js

```diff
@@ -154,7 +154,7 @@ export type Settings = {
   },
   FIRMWARE_DIRECTORY: string,
   FIRMWARE_REPOSITORY_DIRECTORY: string,
-  LOG_REQUESTS: boolean,
+  LOG_LEVEL: string,
```

Comment:
could you make the type more precise, as i see in byan-annotations this is predefined.